ports/zephyr/Kconfig

```diff
@@ -58,6 +58,12 @@ config MICROPY_USB_DEVICE_PID
 	hex "USB PID"
 	default 0x0001
 
+config MICROPY_ADDITIONAL_HEAPS
```

Comment:
Suggest naming this `MICROPY_GC_SPLIT_HEAP` to match the `py/mpconfig.h` option (similar to other Kconfig options in this file).

Reply:
done